default.json

```diff
@@ -74,7 +74,7 @@
       "description": "Group and auto-merge non-major NodeJS dependencies on weekday mornings",
       "matchUpdateTypes": ["minor", "patch", "pin", "digest"],
       "matchDepNames": ["node", "@types/node"],
-      "schedule": ["after 5am and before 4pm every weekday"],
+      "schedule": ["before 5am and after 4pm every weekday"],
```

Comment:
Our other schedules are 
```
"schedule": [
        "after 8pm every weekday",
        "before 8am every weekday",
        "every weekend"
      ],
```
and I'm happy to update everything to a consistent schedule if we want